common/src/main/java/com/linecorp/centraldogma/common/Entry.java

```diff
@@ -174,101 +165,58 @@ public void ifHasContent(Consumer<? super T> consumer) {
     }
 
     /**
-     * Returns the content of this {@link Entry}.
+     * Returns a new {@link Entry} which applied the specified {@link Query} to the {@link #content()}.
```

Comment:
Could you rephrase this a little bit? It's not an entry who applied the query. How about:

```
Applies the specified {@link Query} to the {@link #content()} of this {@link Entry} and returns
the query result.
```